Bond/Bond+UIButton.swift

```diff
@@ -27,13 +27,14 @@
 
 import UIKit
 
-@objc class ButtonDynamicHelper
+class ButtonDynamicHelper: NSObject
```

Comment:
It's probably worth a separate issue, but the Objective-C names for these classes should probably have a class prefix (which is what @objc does in Xcode 7.0b4 and higher I believe)